# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rpcsvc-proto/include-stat-h.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rpcsvc-proto-1.4.3/rpcgen/rpc_main.c.vanilla	2024-04-02 19:01:41.763244617 +0200
+++ rpcsvc-proto-1.4.3/rpcgen/rpc_main.c	2024-04-02 19:03:17.820238034 +0200
@@ -49,6 +49,7 @@
 #include <sys/file.h>
 #include <sys/stat.h>
 #include <sys/wait.h>
+#include <sys/stat.h>
 #include "rpc_parse.h"
 #include "rpc_util.h"
 #include "rpc_scan.h"
